odfuzz/fuzzer.py

```diff
@@ -367,9 +367,16 @@ def generate_body(self,accessible_entity,key_pairs):
                     generated_body = generated_body.strip("\'")
                 except:
                     pass
-                body[prprty._name] = generated_body
+                if Queryable.check_valid_property(prprty._name) == True: 
+                    body[prprty._name] = generated_body
         return body
 
+    def check_valid_property(prprty_name):
```

Comment:
I do not understand the purpose of this function, resp why the "_fc suffix" is creating problem and why it would determine if the property is valid/not valid.

 Is it a real problem on any odata service? it is possible that this created problem only on your service you run the  fuzzer against.

Reply:
the properties with _fc suffix are supposedly UI related fields and are not supposed to part of the service requests, and using them with any value results in errors, in every service they appear, example: **EHFND_TDEF_MOC_SRV** 
Removal seems to be the only fix for now. Jira issue : ODFUZZ-11

Reply:
well the JIRA is only internal. The method name itself is a bit misleading/overgeneral (valid property, based on what?) and no docstring to explain purpose and deep motivation behind it.

Reply:
we discussed this online and found out that this method basically hardcodes what is already configurable trough Restrictions feature - https://github.com/SAP/odfuzz/blob/master/doc/restrictions.rst
Only it will be separate feature work to rewrite the restrictions.py module for usage in DirectBuilder, ideally as standard python dictionary input parameter for DirectBuilder (in place of current null placehodler for compatibility with DispatechBuilder)

Please remove this method entirely, not useful.

Reply:
Removed